pkg/cli/cmd/build.go

```diff
@@ -10,10 +10,23 @@ type BuildFlags struct {
 	DefinitionFile string
 	ConfigDir      string
 	RootBuildDir   string
-	Validate       bool
 }
 
-var BuildArgs BuildFlags
+var (
```

Comment:
This might be excessive, but it feels weird (to me, this is a personal preference sort of thing) to have these defined here even though they are used in another file. Did you consider something like `flags.go`?